src/pluginlist.cpp

```diff
@@ -182,82 +182,85 @@ void PluginList::refresh(const QString& profileName,
       gamePlugins ? gamePlugins->lightPluginsAreSupported() : false;
   const bool overridePluginsAreSupported =
       gamePlugins ? gamePlugins->overridePluginsAreSupported() : false;
+  const bool loadOrderMechanismNone =
+      m_GamePlugin->loadOrderMechanism() == IPluginGame::LoadOrderMechanism::None;
 
   m_CurrentProfile = profileName;
 
-  QStringList availablePlugins;
+  std::vector<std::pair<QString, FileEntryPtr>> availablePlugins;
```

Comment:
This should probably be an `unordered_map`, I do not think insertion would be much slower but that would avoid the expensive `find_if`.

Reply:
Thank you. I have made the requested changes.